README.rst

```diff
@@ -9,6 +9,6 @@ tariochbctools
 ==============
 
 
-Some importers, plugins and price fetchers for the double-entry bookkeeping software `Beancount <http://furius.ca/beancount/>`__.
```

Comment:
should we point this to https://beancount.github.io/ instead?

Reply:
sure, will fix